### PR TITLE
Added Rodryk's Dragon Bridge

### PR DESCRIPTION
**Pull request by ddbb07**
- Added RodryksDragonBridge.esp
- Added patch messages for RodryksDragonBridge.esp
- Added clean messages for RodryksDragonBridge.esp

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -10601,6 +10601,28 @@ plugins:
...
     after: [ 'Requiem.esp' ]
+    
+  - name: 'RodryksDragonBridge.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/42510/' ]
+    msg:
+      - <<: *patch3rdParty
+        subs:
+          - 'Enhanced Lights and FX'
+          - '[ELFX - Rodryk''s Dragon Bridge patch](https://www.nexusmods.com/skyrimspecialedition/mods/43032/)'
+        condition: 'active("EnhancedLightsandFX.esp") and not active("ELFX - Rodryk's Dragon Bridge patch.esp")'
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
        condition: 'active("EnhancedLightsandFX.esp") and not active("ELFX - Rodryk''s Dragon Bridge patch.esp")'
```

File: masterlist.yaml [hunk 2]
```diff
@@ -10601,6 +10601,28 @@ plugins:
...
+          - '[Rodryk''s Dragon Bridge - Great Cities Patch](https://www.nexusmods.com/skyrimspecialedition/mods/42619/)'
+        condition: 'active("The Great Cities - Minor Cities and Towns.esp") and not active("RodryksDragonBridge - GCS Dragon Bridge Patch.esp")'
+    dirty:
+      - <<: *quickClean
+        crc: 0xB73D9C06
+        util: '[SSEEdit v4.0.3](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
+        udr: 1
+    clean:
+      - crc: 0xC110BC98
+        util: 'SSEEdit v4.0.3'
```
Comment (MacSplody) [inline masterlist.yaml]: I'd move to a section with similar mods.
In the `Locations - New Structures & Landmarks` section before 'RREnhanced.esp' would be fine.

image

Comment (sibir-ine) [inline masterlist.yaml]: Since checksums post-QAC can be volatile, we prefer not to add them for dirty plugins.
```suggestion
```

**Review by sibir-ine — DISMISSED** [review]
@ddbb07

**Review by MacSplody — APPROVED** [review]
Perfect, thanks.

**Pull request merged**